deny.toml

```diff
@@ -1,3 +1,13 @@
+[advisories]
+ignore = [
+    # See: https://github.com/MaterializeInc/materialize/issues/6244
+    #
+    # The "vulnerability" here is just a DoS via excessive CPU and memory usage
+    # on large inputs, which is not a particular concern for Materialize, since
+    # the entire system is vulnerable to this class of attack.
```

Comment:
This is in a sense true but could you reword it? It makes it sound like we know we’re fucking up so bad that we’ve stopped bothering to care, rather than this sort of thing just being outside the security/threat/permissions model we’ve chosen to support.

This suggestion is optional, use your judgment. The actual content of the diff LGTM.